### PR TITLE
Relicense the library to MIT/Apache 2.0

### PR DESCRIPTION
**Pull request by wiktor-k**
Due to MIT/Apache 2.0 combination being more widespread in Rust and due to the fact that the library has basically been almost completely rewritten it has been suggested that this is a perfect opportunity to adjust the license.

See: https://rustcrypto.zulipchat.com/#narrow/stream/346919-SSH/topic/Bring.20in.20ssh-agent-lib/near/431145569

[README.md rendered](https://github.com/wiktor-k/ssh-agent-lib/blob/relicense-to-apl-plus-mit/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,16 +70,20 @@ SSH_AUTH_SOCK=\\.\pipe\agent ssh user@example.com
...
-## Contribution
+### Contribution
 
-Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in this crate by you shall be licensed as above, without any additional terms or conditions.
+Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in this crate by you, as defined in the Apache-2.0 license, shall be dual licensed as above, without any additional terms or conditions.
+
+### Note
+
+This library has been forked from [sekey/ssh-agent.rs](https://github.com/sekey/ssh-agent.rs) as the upstream seems not be maintained (at least as of 2022).
+The original library was MIT-licensed but due to this library using MIT/Apache 2.0 any changes that are made over the old bits are re-licensed MIT/Apache 2.0 where the original (c) line was retained in relation "for the x parts derived from y" which identify the forked bits.
```
Comment (wiktor-k) [inline README.md:89]: @baloo could I ask you for a favour in properly writing the thought that pinkforest explained in the zulip thread here? Thanks! :bow:

**Comment by wiktor-k** [comment]
Folks (@jcspencer, @baloo), this has been suggested in the zulip/rust crypto thread and I think it's a perfect opportunity to do it now but since you two contributed significant changes I'd like your explicit acks on it (this is not super-urgent).

**Comment by jcspencer** [comment]
ack from me!

**Comment by baloo** [comment]
Ack from me.

File: README.md (hunk 1 above)
Comment (baloo) [reply README.md:89]: Yup, I'll try to do that. Probably Monday.

File: LICENSE-MIT [hunk 2]
```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2019 SeKey
```
Comment (tarcieri) [inline LICENSE-MIT:3]: If you have completely rewritten the code to where none of the original exists, you can remove their original copyright as well

File: Cargo.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name = "ssh-agent-lib"
 description = "A collection of types for writing custom SSH agents"
 version = "0.4.0"
-license = "MIT"
+license = "MIT OR Apache-2.0"
 authors = [
```
Comment (wiktor-k) [inline Cargo.toml:6]: I think we need to adjust this line too. We're *very* grateful to the original authors :bow: but currently it's @baloo that's maintaining it and @jcspencer who sends occasional PRs. :thinking:

**Comment by wiktor-k** [comment]
Thanks for taking care of this @baloo 🙇‍♂️

**Pull request merged**